onnx/defs/math/defs.cc

```diff
@@ -288,6 +305,24 @@ and output blobs.
     .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
         "Constrain input and output types to float tensors.");
 
+OPERATOR_SCHEMA(ScaledTanh)
```

Comment:
Again, do we really need it as a fused op? Few options: add scalar to the original Tanh or just ask people to use 2 ops. Given that there are so many of the ops added here, I'm a bit worried that we'd overload the spec too much.